core/cap-0021.md

```diff
@@ -297,29 +298,33 @@ fail the second validation, for instance if a `BUMP_SEQUENCE`
 operation made the sequence number invalid.  Whenever a transaction
 fails validation during execution, the `sourceAccount` loses the fee.
 
-Because `PreconditionsV2` specifies multiple pre-conditions, there may
-be multiple reasons why a transaction is invalid.  If any of the
-reasons is permanent---namely the `maxTime` or `maxLedger` has been
-exceeded--then the transaction is rejected with
-`TransactionResultCode` `txTOO_LATE`.  Otherwise, if all the
-unsatisfied preconditions are based on values that could later be
-valid, then the transaction is rejected with `txTOO_EARLY`.  If
-neither of the above conditions holds (`txTOO_EARLY` and `txTOO_LATE`
-do not apply), but one of the `extraSigners` is unsatisfied, then the
-transaction fails with `txBAD_AUTH`.
+Because `PreconditionsV2` specifies multiple pre-conditions, there may be
+multiple reasons why a transaction is invalid. If `extraSigners` contains
+duplicate signers, the transaction is rejected with `txNOT_SUPPORTED` (Note that
+signer overlap between `extraSigners` and `AccountEntry` signers is allowed). If
+the `maxTime` or `maxLedger` has been exceeded--then the transaction is rejected
+with `TransactionResultCode` `txTOO_LATE`. Otherwise, if all the unsatisfied
+preconditions are based on values that could later be valid (the `minTime` or
+`minLedger` have not been reached yet), then the transaction is rejected with
+`txTOO_EARLY`. If the failure is due to `minSeqAge` or `minSeqLedgerGap`, then
```

Comment:
We should specify what happens if `minSeqNum` isn't reached yet, which I assume would still be `txBAD_SEQ`. On read of this it sort of implies it might be `txTOO_EARLY`?

Reply:
Updated